### PR TITLE
refactor: [M3-6273] - MUI v5 Migration - SRC > Features > Lish

### PR DESCRIPTION
**Pull request by jaalah-akamai**
## Description 📝
Refactored `Lish` as part of MUI v5 migration

## Changes  🔄
- `CircleProgress.tsx` - Allow `sx` to be passed in as props
- `ReachTab.tsx` - Remove `!important` that was causing styling issues with `Lish` tabs
- `Lish.tsx` - No need to import css files in `useEffect`, we can import normally.
   - Vertically center the `CircleProgress` when loading
   - Remove use of `Fragment` and return null instead
   - Use `Styled` api instead of `makeStyles`

## Preview 📷
| Before  | After   |
| ------- | ------- |
| Screenshot 2023-10-10 at 10 21 12 AM | Screenshot 2023-10-10 at 10 23 14 AM |
|Screenshot 2023-10-10 at 10 27 34 AM|Screenshot 2023-10-10 at 10 29 30 AM|

| Before  | After   |
| ------- | ------- |
| <video src="https://github.com/linode/manager/assets/125309814/87c01a35-d4e5-4a54-9264-780fab1a9a67" /> | <video src="https://github.com/linode/manager/assets/125309814/ec1c4b17-b14d-4fe1-86b0-29f760508dde" /> |
| <video src="https://github.com/linode/manager/assets/125309814/48c1b0af-0ad3-4dd9-b00a-f32bdb3b59ba" /> | <video src="https://github.com/linode/manager/assets/125309814/53df28da-5364-4216-b69c-bf1ea22d18d2" /> |

### Prerequisites
- Must have a Linode

### Reproduction steps
- Go to linodes detail page
- Open "Launch LISH Console"

### Verification steps 
- Observe hover states for tabs are fixed
- Observe css styles are still working
- Observe loading state is vertically centered (alternatively if you're pulling this down locally, you can just set the `isLoading` condition to `true` to test

## As an Author I have considered 🤔

*Check all that apply*

- [x] 👀 Doing a self review
- [x] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [x] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a changeset
- [x] 🧪 Providing/Improving test coverage
- [x] 🔐 Removing all sensitive information from the code and PR description
- [x] 🚩 Using a feature flag to protect the release
- [x] 👣 Providing comprehensive reproduction steps
- [x] 📑 Providing or updating our documentation
- [x] 🕛 Scheduling a pair reviewing session
- [x] 📱 Providing mobile support
- [x] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/src/features/Lish/Lish.tsx [hunk 1]
```diff
@@ -1,65 +1,27 @@
...
 const AUTH_POLLING_INTERVAL = 2000;
 
-const useStyles = makeStyles((theme: Theme) => ({
-  notFound: {
-    '& h1': {
-      color: '#f4f4f4 !important',
-    },
-    color: '#f4f4f4 !important',
-  },
-  progress: {
```
Comment (jaalah-akamai) [inline packages/manager/src/features/Lish/Lish.tsx:30]: This has no effect whatsoever, so I removed since we're centering this anyways

**Review by bnussman-akamai — APPROVED** [review]

**Comment by tyler-akamai** [comment]
Reproduced error ✅

Ensured that:
- [ ] Hover states for tabs are fixed
- [ ] CSS styles are still working
- [ ] Loading state is vertically centered

**Comment by tyler-akamai** [comment]
Should we left align the terminal text for `Glish` like we have it for `Weblish`?
| Weblish | Glish |
| --- | --- |
| Screenshot 2023-10-10 at 6 38 09 PM | Screenshot 2023-10-10 at 6 39 40 PM |

**Comment by tyler-akamai** [comment]
Also, when you say: "Loading state is vertically centered",

do you mean the loading circle is horizontally centered. I wouldn't consider it vertically centered (Since its closer to the top), but I do like how it looks now.

I just wanted to confirm before checking it off.

**Comment by tyler-akamai** [comment]
Good work so far!

**Comment by jaalah-akamai** [comment]
> do you mean the loading circle progress icon is horizontally centered?

- I was mainly referring to the main spinner that shows upon opening the console, not the ones for each tab. However, I just centered those too.
- Fixed the margin issue

**Comment by tyler-akamai** [comment]
Still seems like there is too much spacing on the left hand side.

<video src="https://github.com/linode/manager/assets/139489745/d53c47e2-37a1-446e-bd4b-ea0b2f5fc59b" />

Not sure if this is a problem that should be addressed in this PR or another. Or maybe its not a problem at all, thoughts?

**Comment by tyler-akamai** [comment]
Confirmed everything works as expected and the text is left aligned ✅

Screenshot 2023-10-11 at 10 04 06 AM

Screenshot 2023-10-11 at 10 03 53 AM

**Review by tyler-akamai — APPROVED** [review]

**Pull request merged**